### PR TITLE
Fix typos in metainfo

### PR DESCRIPTION
**Pull request by kbdharun**
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: data/com.usebottles.bottles.metainfo.xml.in [hunk 1]
```diff
@@ -8,13 +8,13 @@
   <summary>Run Windows Software</summary>
   <developer_name translatable="no">Bottles Contributors</developer_name>
   <description>
-    <p>Bottles lets you run Windows software on Linux, such as applications and games. It introduces a workflow that helps you organize by categorizing each software to your liking. Bottles provides several tools and integrations to help you manage and optimize your applications.</p>
+    <p>Bottles lets you run Windows software on Linux, such as applications and games. It introduces a workflow that helps you organize by categorizing each software to your liking. Bottles provide several tools and integrations to help you manage and optimize your applications.</p>
```
Comment (TheEvilSkeleton) [inline data/com.usebottles.bottles.metainfo.xml.in:11]: "Bottles" in this context is a proper noun in singular, as it's an app called Bottles. In that case, it should be "provides" and not "provide".

That being said, maybe we should reword this to avoid confusion and giving the impression that it's a typo?

Comment (kbdharun) [inline data/com.usebottles.bottles.metainfo.xml.in:11]:
```suggestion
    <p>Bottles lets you run Windows software on Linux, such as applications and games. It introduces a workflow that helps you organize by categorizing each software to your liking. Bottles provides several tools and integrations to help you manage and optimize your applications.</p>
```

Comment (kbdharun) [reply data/com.usebottles.bottles.metainfo.xml.in]: For now, I have reverted the change, I am open to rewording it too.

Comment (TheEvilSkeleton) [reply data/com.usebottles.bottles.metainfo.xml.in]: Now that I think of it, the first sentence starts with "Bottles", followed by a verb in singular. I'm not sure if we can completely reword it without mentioning "Bottles" without sounding bad

Comment (TheEvilSkeleton) [reply data/com.usebottles.bottles.metainfo.xml.in]: I'll keep it as is. I think we've waited long enough.

**Comment by pietrodicaprio** [comment]
Action file fixed @TheEvilSkeleton
